platform/linux-generic/odp_fdserver.c

```diff
@@ -647,6 +647,32 @@ int _odp_fdserver_init_global(void)
 	}
 
 	if (server_pid == 0) { /*child */
+		sigset_t sigset;
+		struct sigaction action;
+
+		sigfillset(&sigset);
+		/* undefined if these are ignored, as per POSIX */
+		sigdelset(&sigset, SIGFPE);
+		sigdelset(&sigset, SIGILL);
+		sigdelset(&sigset, SIGSEGV);
```

Comment:
Maybe the commit comment should be reworded since fdserver is no longer in the same process group as the application. Something about setting signal handling to a known good state regardless of what the application may have done?

Reply:
Yes, makes sense.